doc/howto/json.md

```diff
@@ -0,0 +1,136 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Querying JSON files
+
+In this tutorial, we'll show you how to query JSON with JupySQL and DuckDB.
+
+
+First, let's install the required dependencies:
+
+```{code-cell} ipython3
+:tags: [hide-output]
+
+%pip install jupysql duckdb duckdb-engine rich --quiet
+```
+
+Now, let's generate some data. Note that DuckDB expects your data to contain *one JSON object per line*; this format is called [JSON Lines](https://jsonlines.org/), and it often comes with the `.json`, `.jsonl.gz`, or `.jsonl.bz2` extension.
```

Comment:
This is going to be fixed in the next release, due out in the next fortnight or so

Reply:
awesome, opened an issue so we update the tutorial! https://github.com/ploomber/jupysql/issues/122